hydra-access-controls/lib/hydra-access-controls.rb

```diff
@@ -2,6 +2,7 @@
 require 'active-fedora'
 require 'blacklight'
 require 'cancan'
+require 'action_view' # https://github.com/haml/haml/issues/695
```

Comment:
Since this gem doesn't use haml, but rdf does, should the fix not go into the rdf gem?